bitnami/kafka/templates/scripts-configmap.yaml

```diff
@@ -89,7 +89,7 @@ data:
     export EXTERNAL_ACCESS_PORT={{ .Values.externalAccess.service.port }}
     {{- else if eq .Values.externalAccess.service.type "NodePort" }}
     {{- if .Values.externalAccess.autoDiscovery.enabled }}
-    export EXTERNAL_ACCESS_IP="$(<${SHARED_FILE})"
+    export EXTERNAL_ACCESS_IP="${HOST_IP}"
```

Comment:
I guess we don't need this block anymore in the auto-discovery.sh script:
```
    {{- else if eq .Values.externalAccess.service.type "NodePort" }}
    k8s_svc_node_port() {
        local namespace=${1:?namespace is missing}
        local service=${2:?service is missing}
        local index=${3:-0}
        local node_port="$(kubectl get svc "$service" -n "$namespace" -o jsonpath="{.spec.ports[$index].nodePort}")"
        echo "$node_port"
    }
    k8s_svc_node_port "{{ $releaseNamespace }}" "$SVC_NAME" | tee "$SHARED_FILE"
    {{- end }}
```

Reply:
Actually it is still needed for resolving the PORTs here:
https://github.com/bitnami/charts/blob/457c7302ed0ecb660e77d307effe5cb050fb4fb7/bitnami/kafka/templates/scripts-configmap.yaml#L99

Reply:
You're right!